sed/loader/flash/loader.py

```diff
@@ -220,15 +220,16 @@ def get_elapsed_time(self, fids: Sequence[int] = None, **kwds) -> float | list[f
             raise KeyError(
                 "File statistics missing. Use 'read_dataframe' first.",
             ) from exc
+        time_stamp_alias = self._config["dataframe"].get("time_stamp_alias", "timeStamp")
```

Comment:
`We should add this to the default config. It is also required e.g. in the normalization histogram function for timestamps